frontend/src/metabase/query_builder/components/view/sidebars/DatasetManagementSection/DatasetManagementSection.jsx

```diff
@@ -68,9 +70,32 @@ function DatasetManagementSection({
           icon="model_framed"
           onClick={turnDatasetIntoQuestion}
         >{t`Turn back into a saved question`}</Button>
+        <PLUGIN_MODERATION.QuestionModerationSection
+          question={dataset}
+          reviewBannerClassName="mt1"
+          renderActions={VerifyButton}
+        />
       </SectionContent>
     </div>
   );
 }
 
+VerifyButton.propTypes = {
+  isVerified: PropTypes.bool.isRequired,
+  verifiedIconName: PropTypes.string.isRequired,
+  onVerify: PropTypes.func,
+  testID: PropTypes.string,
+};
+
+function VerifyButton({ isVerified, onVerify, verifiedIconName, testID }) {
```

Comment:
Should it belong to `metabase-enterprise/moderation`?

Reply:
This would mean we'd need to duplicate `DatasetManagementSection's` styles in the plug-in code, so we'll always need to change both. I don't have a clear opinion what's better here: more duplication without a need to introduce render props or less duplication but with some extra complexity. What do you think? Is there a better way maybe?

Reply:
Correct me if I'm wrong, it seems like inside `QuestionModerationSection` we could pass to `ModerationActions` flag `isDataset` to change the copy of the toggle and change styles of what's inside `defaultRender` to look good both for datasets and questions

Reply:
Right, the problem is that we use a [customized `Button`](https://github.com/metabase/metabase/blob/84429c3b8a95dd0f2dc213277756b5dbfde5eb23/frontend/src/metabase/query_builder/components/view/sidebars/DatasetManagementSection/DatasetManagementSection.styled.jsx#L23) for the model's sidebar actions. If we just move this `VerifyButton` to `metabase-enterprise`, we'll need to copy the `Button` styles too, so we'll have them defined in two places (`DatasetManagementSection.styled` and `metabase-enterprise/moderation`). So the button style should be somehow passed to `metabase-enterprise/moderation` to keep the UI consistent

Reply:
Oh, got it. It just looks almost the same, so I thought we could define this custom button as a component in the non-enterprise folder and use it for both DatasetManagementSection's menu items and QuestionModerationSection